scripts/verify-spelling.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TOOL_VERSION="v0.3.4"
```

Comment:
Hi @rikatz . Thanks for adding this spellchecker. It is helpful to have a script to detect spelling errors. Would you have time to add a bit of documentation in the comment section of the script, such as:
- What parameters to use when running the script
- What version of Go is required or mention that you must have Go installed
- Identify the tool (github.com/client9/misspell/)
- Any other suggestions/requirements
Thanks!
/lgtm

Reply:
Added some instructions :) 

hopefully we can generate in a future some common wordlist and use some more complex script to verify also other languages